en/sensor/rpm.md

```diff
@@ -0,0 +1,40 @@
+# ThunderFly TFRPM01 tachometer sensor
```

Comment:
This document will have to be added to en/SUMMARY.md or it will not render (just follow pattern of other docs).

Reply:
Document listing added.